v0/destinations/hs/transform.js

```diff
@@ -218,8 +217,11 @@ async function processSingleMessage(message, destination) {
   return response;
 }
 
-function process(event) {
-  return processSingleMessage(event.message, event.destination);
+// has been deprecated - using routerTransform for both the versions
+async function process(event) {
+  // no. of calls for properties endpoint remains the same
+  const propertyMap = await getProperties(event.destination);
+  return processSingleMessage(event.message, event.destination, propertyMap);
 }
```

Comment:
Although not really a bad change, I would revert these changes and just send `undefined` in place of `propertyMap` to align with my comment below.

Reply:
sent undefined. The code looks cleaner this way.